src/librustc_resolve/diagnostics.rs

```diff
@@ -23,8 +23,9 @@ use crate::imports::{Import, ImportKind, ImportResolver};
 use crate::path_names_to_string;
 use crate::{AmbiguityError, AmbiguityErrorMisc, AmbiguityKind};
 use crate::{
-    BindingError, CrateLint, HasGenericParams, MacroRulesScope, Module, ModuleOrUniformRoot,
+	CrateLint, HasGenericParams, MacroRulesScope, Module, ModuleOrUniformRoot,
```

Comment:
```suggestion
	CrateLint, HasGenericParams, MacroRulesScope, Module, ModuleOrUniformRoot,
```

Use whitespaces instead of hard tabs.